src/commands/local.ts

```diff
@@ -629,16 +632,28 @@ async function startProcesses(
 
             await writeToFile(
                 path.join(tmpFolder),
-                "local_function_wrapper.mjs",
-                getLocalFunctionWrapperCode(functionInfo.handler, functionInfo.entry),
+                `local_function_wrapper.${entryFileFunctionMap[functionInfo.language as keyof typeof entryFileFunctionMap].split(".")[1]}`,
```

Comment:
This is kinda ugly :)) For readability and because you use it in multiple places, I would create a helper function that creates filenames like `local_function_wrapper.mjs`, `local_function_wrapper,py` or `index-random.mjs`:

```
function getFunctionEntryFilename(language: Language, filename: string): string {
    const extension = entryFileFunctionMap[language as keyof typeof entryFileFunctionMap].split(".")[1];
    return `${filename}.${extension}`;
}
```